server/src/main/resources/application.yml

```diff
@@ -5,6 +5,7 @@ lighter:
   frontend-path: file:${FRONTEND_PATH:../frontend/build}
   py-gateway-port: 25333
   url: http://lighter.spark:8080
+  lighter-zombie-interval: 30m
```

Comment:
Are you sure? Then env variable would be LIGHTER_LIGHTER….

Reply:
thanks